### PR TITLE
Optimize Ladder Index

### PR DESCRIPTION
**Pull request by rohsyl**

### CODE REVIEW

File: cncnet-api/app/Http/Controllers/ApiLadderController.php [hunk 1]
```diff
@@ -156,11 +153,8 @@ public function saveLadderResult($file, $ladderId, $gameId, $playerId, $pingSent
         if ($ladderId == 8 || $ladderId == 1) //toggle achievements on for Blitz and YR
             $this->updateAchievements($playerId, $history->ladder, $stats);
 
-        // Toggling off auto-wash, TODO only wash games where both players submitted a game report. 
-        // If only one player submitted a game report, don't wash
-
-        // if ($gameReport->best_report == true && $gameReport->duration == 3 && $gameReport->fps == 33)
-            // $this->adminService->doWashGame($gameReport->game_id, "ladder-auto-wash");
+        if ($gameReport->best_report == true && $gameReport->duration == 3 && $gameReport->fps == 33)
+            $this->adminService->doWashGame($gameReport->game_id, "ladder-auto-wash");
```
Comment (alexp8) [inline cncnet-api/app/Http/Controllers/ApiLadderController.php:157]: this intended? @rohsyl 

Looks like a merge mistake, lines 156/157 should be commented out / removed

Comment (alexp8) [reply cncnet-api/app/Http/Controllers/ApiLadderController.php:157]: remove or comment out

```
 if ($gameReport->best_report == true && $gameReport->duration == 3 && $gameReport->fps == 33)
            $this->adminService->doWashGame($gameReport->game_id, "ladder-auto-wash");
            ```

File: cncnet-api/app/Models/Ban.php [hunk 2]
```diff
@@ -54,7 +54,7 @@ public function ip()
 
     public static function unstartedBanTime()
     {
-        return Carbon::now();
+        return Carbon::create(0, 0, 0, 0, 0, 0);
```
Comment (alexp8) [inline cncnet-api/app/Models/Ban.php:57]: is this change intended?

Comment (rohsyl) [reply cncnet-api/app/Models/Ban.php]: fixed

File: cncnet-api/app/Http/Controllers/ApiLadderController.php (hunk 1 above)
Comment (rohsyl) [reply cncnet-api/app/Http/Controllers/ApiLadderController.php]: fixed

**Review by rohsyl — COMMENTED** [review]
Refactored many blade components. Olds need to be removed once we are sure they are but used anymore.

**Review by alexp8 — APPROVED** [review]
